src/resources/views/quiz.blade.php

```diff
@@ -12,7 +12,7 @@
             <img src="/images/{{$question->image}}" alt="">
 
             <ul class="ul" id="question{{$question->id}}_choices">
-                @foreach ($choices[$loop->index] as $choice) 
+                @foreach ($question->choices as $choice) 
```

Comment:
```suggestion
                @foreach ($question->choices as $choice) 
```
n+1問題 (子孫要素をforeachを実行する回数分SQLが発行されて処理が重くなる) が発生するのでコントローラー側でwith等を利用してください
`参考記事`
https://migisanblog.com/laravel-with-load-n/